CHANGELOG.md

```diff
@@ -75,6 +78,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Clarified that back-ends not supporting pagination will return all resources.
 - Clarified how `from_parameter` is resolved in case no value is given.
 - Clarified `GET .../logs` endpoint behaviour.
+- Clarify difference between STAC specification and STAC API.
+- Clarify that a copy of the STAC Item should be part of the assets in a batch job download.
```

Comment:
bit of nitpicking: I read the "should" as "MUST", while the spec diff below states "RECOMMENDED".
Maybe change to something like

> Added recommendation to include a link to the STAC Item in assets of batch job download

Reply:
should is never a must, but will change:

```suggestion
- Clarify that a copy of the STAC Item is recommended to be part of the assets in a batch job download.
```

Reply:
yes I know, it's just that kind of terminology is less strictly or explicitly defined in the context of a CHANGELOG listing